mono/metadata/assembly.c

```diff
@@ -133,36 +135,140 @@ static const AssemblyVersionMap framework_assemblies [] = {
 	{"Novell.Directory.Ldap", 0},
 	{"PEAPI", 0},
 	{"System", 0},
+	{"System.AppContext", 0},
```

Comment:
AFAIK this was a hack not to list Facades assemblies in framework_assemblies. If it cannot remap the dll we still have some remapping issue somewhere